kbase-extension/static/kbase/templates/data_staging/file_path.html

```diff
@@ -1,4 +1,12 @@
-<button id="refresh" class="btn btn-xs btn-default">
+<span class="kb-data-stagingarea">Staging Area</span>
```

Comment:
I think the staging area should be contained within a `<section></section>` element, and the span should in realty he some level of a heading i.e. <h2> <h3> ect. I can't find where the rendered wrapper element for the staging area is created. The rendered wrapper of the staging area is 
```
<div id="kb-data-staging-table_wrapper" class="dataTables_wrapper form-inline dt-bootstrap no-footer">
   ...
</div>
```

Reply:
I could be wrong though. That is just my gut feeling.

Reply:
I believe the dataTables plugin generated this. I honestly don't know how to answer your above comment except by noting that those elements are inline block and would cause line breaks, which means it wouldn't match the design

Reply:
Yeah, that's autogenerated by dataTables. all the _wrapper stuff is like that.